charts/linkerd-control-plane/values.yaml

```diff
@@ -109,6 +109,12 @@ proxy:
   # -- Maximum time allowed for the proxy to establish an inbound TCP
   # connection
   inboundConnectTimeout: 100ms
+  # -- Maximum time allowed before an unused outbound discovery result
+  # is evicted from the cache
+  outboundDiscoveryCacheUnusedTimeout: "5s"
+  # -- Maximum time allowed before an unused inbound discovery result
+  # is evicted from the cache
+  inboundDiscoveryCacheUnusedTimeout: "90s"
```

Comment:
Naming nit: Why the departure from the convention used in the other timeout vars, that reflect the env vars names, and not simply using `outboundDiscoveryIdleTimeout` and `inboundDiscoveryIdleTimeout`?

Reply:
I think the two are a bit ambiguous, it's not immediately obvious to someone not familiar with proxy internals that they're actually used for the cache timeouts. They could be misinterpreted as being connection/request timeouts.